ManatiCR/manatime
# 67: Edit and remove default tasks.

**Steps to test**

- Add some default tasks in /admin/config/manatime/default-tasks.
- Go to /node/add/projects and check if the new default tasks has been added.
- Check if the actions edit and remove run successfully.

[issue](https://github.com/ManatiCR/manatime/issues/67)
